README.md

```diff
@@ -10,14 +10,20 @@ Setup (required Node 4.x)
 
     yarn
 
-Run a dev server
+    yarn postinstall
```

Comment:
I want to add bower install command. But there are the line at the scripts at the package.json: "postinstall": "bower install && gulp"...